src/actions/index.js

```diff
@@ -18,3 +19,9 @@ export const login = formData => async (dispatch) => {
     return message;
   }
 };
+export const socialAuth = (token) => {
```

Comment:
Please abstract this. There should be separate files for auth.

Reply:
resolved